sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcSQLQueryBuilder.scala

```diff
@@ -81,17 +81,20 @@ class JdbcSQLQueryBuilder(dialect: JdbcDialect, options: JDBCOptions) {
   /**
    * Constructs the WHERE clause that following dialect's SQL syntax.
    */
-  def withPredicates(predicates: Array[Predicate], part: JDBCPartition): JdbcSQLQueryBuilder = {
+  def withPredicates(predicates: Array[Predicate],
```

Comment:
please start these arguments on the next line, indented by +4 spaces each.

Reply:
Done